src/background.js

```diff
@@ -238,38 +243,56 @@ function createTray() {
 async function downloadWalletRelease(releaseUrl, version) {
   status = STATUS.WAIT
   loadUrl(STATUS.WAIT)
-
-  await sleep(5000)
-
+  const file = path.join(
+    SHEIKAH_PATH,
+    `witnet-0.9.2-x86_64-apple-darwin.tar.gz`,
```

Comment:
Use  `witnet-0.9.2-${arch}-apple-${arch}.tar.gz` for mac and `witnet-release-${arch}-${platform}.tar.gz` for linux